react/react1/week2/to-do-list/src/App.css

```diff
@@ -0,0 +1,14 @@
+.Container {
+  margin: 20px;
+}
+
+.listItems {
+  font-size: 16px;
+  line-height: 2em;
+  list-style-type: none;
+}
+
+.style-checked {
```

Comment:
Same comment is valid about this.

Above you have `.listItems` which uses camelCase, while here, you use `.style-checked` which uses kebab-case. Consistency is very important, both for someone reviewing this, but also for future self :) You should pick one and have everything be similar.